department-of-veterans-affairs/abd-vro
# add /claim-linker endpoint - log VBMS and VA.gov claim id's

Depends on:
https://github.com/department-of-veterans-affairs/abd-vro/pull/2407

## What was the problem?

VA.gov claim id is collected from /classifier, but VBMS id for the claim is not saved or linked to the va.gov claim presently.


Associated tickets or Slack threads:
- https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/152

## How does this fix it?[^1]

- added endpoint for logging va.gov and vbms claim ids together `/claim-linker`


## How to test this PR
- `pytest`


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
